src/cli.js

```diff
@@ -208,8 +213,9 @@ const main = () => {
         });
     } else {
         console.info(`Watching ${filesPattern} ...\n`);
-
-        const watcher = chokidar.watch(filesPattern);
+        
+        const chokidarOptions = argv.i ? {ignored: argv.i} : null;
```

Comment:
the variable is not being used in watch below. Do you forget to use it ?

Reply:
Fixed

Reply:
Thank you !